client/lib/pages/plant-list.dart

```diff
@@ -77,8 +80,8 @@ class PlantListItem extends StatelessWidget {
             children: <Widget>[
               Expanded(
                 flex: 4,
-                child: Image.asset(
-                  plant.imageName,
+                child: Image.file(
+                  File(plant.imageName),
```

Comment:
`plant.imageName` is null after pressing the 'voeg toe' button ,causing the app to show an error.

![2020-05-20 17_27_11-ONEPLUS A6003](https://user-images.githubusercontent.com/22012835/82465521-aa9bf500-9abf-11ea-804d-b3cdee4b6e8a.png)